packages/noya-utils/src/getIncrementedName.ts

```diff
@@ -0,0 +1,12 @@
+const numberSuffixRegExp = /(\s\d+)$/;
+
+export function getIncrementedName(name: string): string {
```

Comment:
What you have works and has all the right logic, but I think we can simplify it by including the "prefix" (non-number) part of the string in the regex. Does this work?

```ts
const numberSuffixRegExp = /(.*?)(\s\d+)?$/
  
function getIncrementedName(name: string): string {
  const [, prefix, number] = name.match(numberSuffixRegExp)
  const parsedNumber = (number ? parseInt(number) : 1)
  return `${prefix} ${parsedNumber + 1}`
}
```

Reply:
This is the result of using that regexp 😅 
![image](https://user-images.githubusercontent.com/16924758/115602231-520f4200-a2a4-11eb-9cf7-5c604f6ad7dd.png)

Reply:
Oh I just applied the full function and it works. I'll figure out the other part for the name 🤔 I think I Can use a list of the names that are the same